local/protocol/local.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+// TODO(dbentley): integrate this into our go build
```

Comment:
Any particular reason for choosing this over thrift? Is it easier to work with when making something simple?